swagger.json

```diff
@@ -725,6 +725,48 @@
                         }
                     }
                 }
+            },
+            "get": {
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/accessTokenParam"
+                    },
+                    {
+                        "name": "group_id",
+                        "required": false,
+                        "in": "query",
+                        "type": "integer",
+                        "format": "int64",
+                        "description": "Optional Group ID to query for.",
```

Comment:
In the ApiHandler, we'll either use this query param passed in from the request, or we'll default to the one group that's usually associated with all orgs.